l10n_jp_hr_employee/tests/test_l10n_jp_hr_employee.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+from odoo.exceptions import ValidationError
+
+
+class L10nJpHrEmployee(common.TransactionCase):
+
+    def setUp(self):
+        super(L10nJpHrEmployee, self).setUp()
+        # Demo user
+        self.test_user = self.env.ref('base.user_demo')
+        # Employee record that linked to Demo user
+        self.test_employee = self.env.ref('hr.employee_qdp')
+        # Japan
+        self.japan = self.env.ref('base.jp')
+
+    def test_00_create_private_information(self):
```

Comment:
@TimLai125 This test looks strange to me. We want to confirm that record can be create with no error, right?  We should also minimize the fields to use here, only including required fields. The purpose of the test gets diluted, otherwise.

Reply:
@yostashiro I am still working on the test